src/app/rpmostree-builtin-ex.c

```diff
@@ -59,16 +63,26 @@ internals_option_context_new_with_commands (void)
 }
 
 int
-rpmostree_builtin_internals (int argc, char **argv, GCancellable *cancellable, GError **error)
+rpmostree_builtin_ex (int argc, char **argv,
+                      RpmOstreeCommandInvocation *invocation,
+                      GCancellable *cancellable, GError **error)
 {
+  g_autoptr(GOptionContext) context = ex_option_context_new_with_commands ();
   RpmOstreeCommand *subcommand;
   const char *subcommand_name = NULL;
   g_autofree char *prgname = NULL;
   int exit_status = EXIT_SUCCESS;
 
   subcommand_name = rpmostree_subcommand_parse (&argc, argv);
-
-  subcommand = internals_subcommands;
+  if (!rpmostree_option_context_parse (context, NULL,
```

Comment:
> Otherwise, e.g. rpm-ostree ex unpack --help will print the help message for rpm-ostree ex rather than the unpack subcommand. 

Looks like this is also an issue with the `container` subcommands.